source/class/qx/tool/cli/Watch.js

```diff
@@ -131,7 +131,9 @@ qx.Class.define("qx.tool.cli.Watch", {
           });
         });
       }))).then(() => {
-        qx.tool.compiler.Console.debug(`DEBUG: confirmed=${JSON.stringify(confirmed)}`);
+        if (this.isDebug()) {
```

Comment:
do we need this?  ie, shouldn't that be in the `qx.tool.compiler.Console.debug` code?

Reply:
It's your own code. See line 109.

Reply:
i meant shouldn't the `if (this.isDebug())` be in the `Console.debug` function?  Otherwise it would have to be applied to every call to Console.debug...